### PR TITLE
task_6626_init_ui_handle_detail_collections

### PR DESCRIPTION
**Pull request by QNFC**
screenshot_2019-02-22-09-05-15-87
init_ui_handle_detail_collections
Link redmine: https://edu-redmine.framgia.vn/issues/6626

### CODE REVIEW

**Comment by QNFC** [comment]
ready

File: app/src/main/java/com/framgia/my_editor_02/screen/collectionsDetail/CollectionDetailAdapter.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            mViewModel = new ItemCollectionDetailViewModel(itemRecyclerViewClick);
+            mBinding.setViewModel(mViewModel);
+        }
+
+        void bindData(Photo photo) {
+            mViewModel.setCollectionDetail(photo);
+            mBinding.executePendingBindings();
+        }
+
+
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/collectionsDetail/CollectionDetailAdapter.java]: remove blank 79-80

File: app/src/main/java/com/framgia/my_editor_02/screen/collectionsDetail/ItemCollectionDetailViewModel.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import android.view.View;
+import com.framgia.my_editor_02.data.model.Photo;
+import com.framgia.my_editor_02.utils.OnItemRecyclerViewClick;
+import java.util.Objects;
+
+public class ItemCollectionDetailViewModel extends BaseObservable {
+    private ObservableField<Photo> mPhotoObservableField;
+    private OnItemRecyclerViewClick<Photo> mItemRecyclerViewClick;
+
+    ItemCollectionDetailViewModel(OnItemRecyclerViewClick<Photo> itemRecyclerViewClick) {
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/collectionsDetail/ItemCollectionDetailViewModel.java:14]:
```suggestion
    ItemCollectionDetailViewModel(OnItemRecyclerViewListener<Photo> itemRecyclerViewClick) {
```

File: app/src/main/res/layout/fragment_detail_collections.xml [hunk 3]
```diff
@@ -14,19 +16,106 @@
...
+                    bind:circleImage="@{viewModel.imageProfile}"
+                    />
+
+                <TextView
+                    android:id="@+id/textViewName"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:text="@{viewModel.collection.title}"
+                    android:textColor="#A3070606"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_detail_collections.xml]: hard code

File: app/src/main/res/layout/fragment_detail_collections.xml [hunk 4]
```diff
@@ -14,19 +16,106 @@
...
+                    app:layout_constraintTop_toBottomOf="@id/imageViewUser"
+                    />
+
+                <TextView
+                    android:id="@+id/textViewTitle"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:text="@{viewModel.collection.title}"
+                    android:textColor="#000"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_detail_collections.xml]: hard code

File: app/src/main/res/layout/fragment_detail_collections.xml [hunk 5]
```diff
@@ -14,19 +16,106 @@
...
+                    app:layout_constraintTop_toBottomOf="@id/textViewName"
+                    />
+
+                <TextView
+                    android:id="@+id/textViewTotal"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:gravity="center"
+                    android:text="@{viewModel.totalPhoto}"
+                    android:textColor="#A3070606"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_detail_collections.xml]: hard code

File: app/src/main/res/layout/item_detail_collections.xml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+            />
+
+        <TextView
+            android:id="@+id/textViewContainer"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_detail_collections.xml]: remove blank

**Comment by QNFC** [comment]
ready

**Comment by tuanlvt-1117** [comment]
commented

**Comment by QNFC** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**